bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Container.java

```diff
@@ -92,6 +92,15 @@ default String getAppliedCssClasses() {
         return null;
     }
 
+    /**
+     * @return returns the role attribute that will be applied on the container.
+     */
+    @Nullable
+    @JsonIgnore
+    default String getRoleAttribute() {
```

Comment:
Any change in interface means changes in package-info, please also update java doc

Reply:
Java doc is already updated. Updated the package-info.

Reply:
Please add the following annotations in the java doc

```
  * @since com.adobe.cq.forms.core.components.models.form 5.4.3
  ```
  
The point of updating package-info is to make sure that the java doc align to the package version

Reply:
Added this!